land_vision/get_images.py

```diff
@@ -0,0 +1,51 @@
+import cv2
+import json
+import requests
+import numpy as np
+import matplotlib.pyplot as plt
+
+from mapbox import mapbox_polygon as _mapbox_polygon
+  
+
+
+def get_mapbox_static_image(latitude, longitude, zoom=15, width=400, height=300, style='streets-v11'):
+
+    base_url = f'https://api.mapbox.com/styles/v4/mapbox.{style}/static/{longitude},{latitude},{zoom}/{width}x{height}'
+    access_token = 'pk.eyJ1IjoibHVrZW0xMjMiLCJhIjoiY2ttc2U4NHkzMGd5bzJ2bG1neTBvdnRqOCJ9.o4fk2TYxHvDtEBu4ddgrmA'
+    
+    params = {
+        'access_token': access_token,
+    }
+    response = requests.get(base_url, params=params)
+
+    if response.status_code == 200:
+        return response.content
+    else:
+        print(f"Error {response.status_code}: {response.text}")
+        return None
+
+def get_mapbox_naip_image(lngLat, zoom, tileType = 'satellite', pixelsPerTile = 512):
```

Comment:
This also probably can use the mapbox_polygon.py function(s), possibly with some updates for this use case.

Reply:
Yes, I could change the mapbox_polygon func to accept both street view and naip images, may be in another pull request since as said above, we don't need get_images.py to run tree detection.